### PR TITLE
Diana navbar

### PR DESCRIPTION
**Pull request by dgarciaperales**
Fixed: The "about" and "contact" page, hover functionality, h2 overlapping/conflicting styling, conflicting footer inconsistencies, and font inconsistencies.

### CODE REVIEW

File: src/styles/navstyles.css [hunk 1]
```diff
@@ -16,11 +16,49 @@
...
   body {
-    font-family: "Segoe UI Variable Display", "Segoe UI", -apple-system, Roboto,
-      sans-serif, serif;
+    font-family: "Roboto Slab", serif;
     margin: 0;
-    /* overflow-x: hidden; */
+    font-weight: normal;
   }
+
+  header {
```
Comment (AmaraWilliams) [inline src/styles/navstyles.css:24]: Could you add a comment for all the styles you have in here? That way it's easier to understand what parts of the site they change without having to go back and forth between the javascript and css files. Same kind of comment you have on line 51 is what I'm looking for.

**Review by AmaraWilliams — APPROVED** [review]
Great job fixing the functionality and the padding on the header! I'll approve it, however I would appreciate if you added comments and added a more descriptive pull request description that includes a before and after picture. You can follow Josh's PR template or simply take a look at previous PRs that got merged.

File: src/components/newnav.js [hunk 2]
```diff
@@ -56,13 +62,25 @@ function Newnav({ transparent }) {
               alt="Empathy Bytes Logo"
             />
             </Link>
-            <h1 className="nav-title"> Empathy Bytes</h1>
+            <h2 className="body"> Empathy Bytes</h2>
```
Comment (joshrsamuel) [inline src/components/newnav.js:65]: Pr looks great! Only thing is, I think i'd be great if users could not only click on the logo, but also the name of our VIP to return to the hime page. This should be as easy as wrapping the h2 in a link as well. 

```
<Link to={"/"}>
            <h2 className="body"> Empathy Bytes</h2>
 </Link>
```

**Review by joshrsamuel — CHANGES_REQUESTED** [review]

**Review by joshrsamuel — APPROVED** [review]
Great work Diana! Thank you for implementing the changes I suggested. The only issue I still see is the scrolling issue we saw on Tiffany and Xavier's tickets, but I think this will be solved with another ticket/issue.

**Pull request merged**